dev-bin/release.sh

```diff
@@ -62,8 +99,11 @@ fi
 mvn release:clean
 mvn release:prepare -DreleaseVersion="$version" -Dtag="$tag"
 mvn release:perform
+rm -fr ".gh-pages/doc/$tag"
+cp -r target/apidocs ".gh-pages/doc/$tag"
+ln -fs "$tag" .gh-pages/doc/latest
```

Comment:
Do we need to commit and push in `.gh-pages` too?

Reply:
Thanks for noticing this!